.github/workflows/tests.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - master
+      - '*-maintenance'
```

Comment:
I think we can skip that here (if we ever need multiple branches I'd go for the 1.0.x style like in indico)